src/templates/app/_layout/footer.jsx

```diff
@@ -190,10 +190,10 @@ const FooterNormal = () => (
-                            '<a href=\'https://secure.gamblingcommission.gov.uk/gccustomweb/PublicRegister/PRSearch.aspx?ExternalAccountId=39172\' target=\'_blank\' rel=\'noopener noreferrer\'>',
+                            'This website is offered in the EU, for financial products, by Binary Investments (Europe) Ltd., Mompalao Building, Suite 2, Tower Road, Msida MSD1825, Malta, licenced and regulated as a Category 3 Investment Services provider by the Malta Financial Services Authority (license no. IS/70156). Volatility Indices products are offered in the UK and Isle of Man by Binary (IOM) Ltd., First Floor, Millennium House, Victoria Road, Douglas, IM2 4RW, Isle of Man, British Isles, licenced and regulated by (1) the Gambling Supervision Commission in the Isle of Man, British Isles, licence issued on 31 August 2012, and for UK clients by (2) the UK Gambling Commission. In the rest of the EU, Volatility Indices products are offered by Binary (Europe) Ltd., Mompalao Building, Suite [_1]2, Tower Road, Msida[_2] MSD1825, Malta, licenced and regulated by (1) the Malta Gaming Authority in Malta, licence no MGA/CL2/118/2000, 26th May 2015 and for UK clients by (2) the UK Gambling Commission, and for Irish clients by (3) the Revenue Commissioners in Ireland, Remote Bookmaker\'s Licence issued on 1 July 2017 (licence no. 1010285). View further [_3]Regulatory Information[_2]. This website\'s services are not made available in certain countries such as the USA, Costa Rica, Hong Kong, or to persons under age 18.',
+                            '<a href=\'https://maps.google.com/?q=2,+Tower+Road,+Msida&entry=gmail&source=g\' target=\'_blank\' rel=\'noopener noreferrer\'>',
```

Comment:
what is this broken maps.google.com link for?.. please remove it all. It doesn't even point to the right Tower Road.

Reply:
The map link is in the email
![image](https://user-images.githubusercontent.com/4541182/36956517-c02aecfa-2069-11e8-9578-c6a9515d64fe.png)

Reply:
done